content/blog/2019/mesh-federation/index.md

```diff
@@ -0,0 +1,1816 @@
+---
+title: Mesh Federation
+subtitle: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control
+description: Connect multiple service meshes ad hoc, with limited service exposure and strict cross-cluster access control.
+publishdate: 2019-08-12
+attribution: Vadim Eisenberg (IBM)
+keywords: [traffic-management,multicluster,security,gateway,tls]
+---
+
+Managing applications across multiple Kubernetes clusters is currently one of the hottest topics in Kubernetes and Istio
+communities. You may need to split your applications between multiple clusters for different reasons, for example:
+
+-  scalability limits of a single cluster
+-  separation of production/dev/test/staging environments
+-  different security and compliance requirements for different parts of the application
+-  running parts of the application on multiple public clouds, and on a private cloud
+
+Currently there are [three patterns](/docs/setup/kubernetes/install/multicluster/) in Istio to create a
+[multicluster service mesh](/docs/concepts/multicluster-deployments/#multicluster-service-mesh).
+A _multicluster service mesh_ is a single _logical_ service mesh, spread among multiple Kubernetes clusters.
+
+Such a service mesh has the following characteristics:
```

Comment:
I feel that this is simply the definition of a service mesh, not a particular type of service mesh. They way this is stated is a bit confusing since it seems like this is somehow different from other types of meshes. It might be better to start out by defining the properties of a service mesh and then go on to show how these properties can be maintained with a multi-cluster topology.